### DISCUSSION TITLE
Valet hangs when initially loading local site on macOS 12

### DISCUSSION THREAD

**Discussion by estin92** [laravel/valet#1176] [Q&A] [answered]
- Valet Version: `2.18.3`
- PHP Version: `8.0.14|8.1.1`

### Description:
I've recently upgrade to an M1 Max chip machine. Since the change, valet hangs when loading local sites. Once that first page has loaded, the experience resumes to normal for a short period. When breaking off to work on some features though and later returning to the browser, the page stalls when trying to interact with it again.

I've even tried downgrading my valet version to `2.13` and re-running `valet install` which was what was on the Intel Mac, but I get the same experience. Wondering (hoping) if anyone has experienced similar and has a workaround? I've included a screen grab below of a fresh Laravel installation that is loaded through valet. It would appear the DNS look-up is what takes the time but not sure why the experience is so volatile between painfully slow vs the normal valet speeds?

Both machines are using `dnsmasq` `2.86`

<img width="1709" alt="Screenshot 2022-01-05 at 20 49 43" src="https://user-images.githubusercontent.com/11060245/148288527-3c54db8c-0d62-436e-b919-68b9d5883ca9.png">

### Steps To Reproduce:
1. Follow installation instructions
2. Link the website either via `valet park` in the parent directory or `valet link` within a specific project
3. Attempt to load in browser

### Diagnosis

<details>
<summary>sw_vers</summary>
<pre>ProductName:	macOS
ProductVersion:	12.1
BuildVersion:	21C52</pre>
</details>
<details>
<summary>valet --version</summary>
<pre>Laravel Valet 2.18.3</pre>
</details>
<details>
<summary>cat ~/.config/valet/config.json</summary>
<pre>{
    "tld": "test",
    "loopback": "127.0.0.1",
    "paths": [
        "/Users/xxx/.config/valet/Sites",
        "/Users/xxx/Sites"
    ]
}</pre>
</details>
<details>
<summary>cat ~/.composer/composer.json</summary>
<pre>{
    "require": {
        "laravel/valet": "^2.18",
        "laravel/installer": "^4.2"
    }
}</pre>
</details>
<details>
<summary>composer global diagnose</summary>
<pre>Changed current directory to /Users/xxx/.composer
Checking composer.json: WARNING
No license specified, it is recommended to do so. For closed-source software you may use "proprietary" as license.
Checking platform settings: OK
Checking git settings: OK
Checking http connectivity to packagist: OK
Checking https connectivity to packagist: OK
Checking github.com oauth access: OK
Checking disk free space: OK
Checking pubkeys: 
Tags Public Key Fingerprint: 57815BA2 7E54DC31 7ECC7CC5 573090D0  87719BA6 8F3BB723 4E5D42D0 84A14642
Dev Public Key Fingerprint: 4AC45767 E5EC2265 2F0C1167 CBBB8A2B  0C708369 153E328C AD90147D AFE50952
OK
Checking composer version: You are not running the latest stable version, run `composer self-update` to update (2.1.14 => 2.2.3)
Composer version: 2.1.14
PHP version: 8.0.14
PHP binary path: /usr/local/Cellar/php@8.0/8.0.14/bin/php
OpenSSL version: OpenSSL 1.1.1m  14 Dec 2021
cURL version: 7.80.0 libz 1.2.11 ssl (SecureTransport) OpenSSL/1.1.1m
zip: extension present, unzip present, 7-Zip not available</pre>
</details>
<details>
<summary>composer global outdated</summary>
<pre>Changed current directory to /Users/xxx/.composer
Legend:
! patch or minor release available - update recommended
~ major release available - update possible
illuminate/container             v8.76.2 ! v8.78.1 The Illuminate Container package.
illuminate/contracts             v8.76.2 ! v8.78.1 The Illuminate Contracts package.
laravel/valet                    v2.18.3 ! v2.18.4 A more enjoyable local development experience for Mac.
psr/container                    1.1.2   ~ 2.0.2   Common Container Interface (PHP FIG PSR-11)
psr/simple-cache                 1.0.1   ~ 3.0.0   Common interfaces for simple caching
symfony/console                  v5.4.1  ~ v6.0.2  Eases the creation of beautiful and testable command line interfaces
symfony/deprecation-contracts    v2.5.0  ~ v3.0.0  A generic function and convention to trigger deprecation notices
symfony/polyfill-ctype           v1.23.0 ! v1.24.0 Symfony polyfill for ctype functions
symfony/polyfill-intl-grapheme   v1.23.1 ! v1.24.0 Symfony polyfill for intl's grapheme_* functions
symfony/polyfill-intl-normalizer v1.23.0 ! v1.24.0 Symfony polyfill for intl's Normalizer class and related functions
symfony/polyfill-mbstring        v1.23.1 ! v1.24.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php73           v1.23.0 ! v1.24.0 Symfony polyfill backporting some PHP 7.3+ features to lower PHP versions
symfony/polyfill-php80           v1.23.1 ! v1.24.0 Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions
symfony/process                  v5.4.0  ~ v6.0.2  Executes commands in sub-processes
symfony/service-contracts        v2.5.0  ~ v3.0.0  Generic abstractions related to writing services
symfony/string                   v5.4.0  ~ v6.0.2  Provides an object-oriented API to strings and deals with bytes, UTF-8 code points and grapheme clusters in a unified way
symfony/var-dumper               v5.4.1  ~ v6.0.2  Provides mechanisms for walking through any arbitrary PHP variable
tightenco/collect                v8.76.1 ! 8.78.0  Collect - Illuminate Collections as a separate package.</pre>
</details>
<details>
<summary>ls -al /etc/sudoers.d/</summary>
<pre>total 16
drwxr-xr-x   4 root  wheel   128 17 Dec 21:22 .
drwxr-xr-x  82 root  wheel  2624  5 Jan 20:10 ..
-rw-r--r--   1 root  wheel    83 17 Dec 21:22 brew
-rw-r--r--   1 root  wheel    86 17 Dec 21:22 valet</pre>
</details>
<details>
<summary>brew config</summary>
<pre>HOMEBREW_VERSION: 3.3.9
ORIGIN: https://github.com/Homebrew/brew
HEAD: 96137bc19e68398ebbb7033379df288cd8b9a3f9
Last commit: 3 weeks ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 5ddc309f5f18b95d23aef9d3557ca067ec015172
Core tap last commit: 3 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit westmere
Clang: 13.0.0 build 1300
Git: 2.32.0 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.1-x86_64
CLT: 13.2.0.0.1.1638488800
Xcode: N/A
Rosetta 2: true</pre>
</details>
<details>
<summary>brew services list</summary>
<pre>Name         Status  User       File
dnsmasq      none root       
mariadb@10.5 started xxx ~/Library/LaunchAgents/homebrew.mxcl.mariadb@10.5.plist
nginx        none root       
php          started xxx ~/Library/LaunchAgents/homebrew.mxcl.php.plist
php@7.4      none            
php@8.0      none root</pre>
</details>
<details>
<summary>brew list --formula --versions | grep -E "(php|nginx|dnsmasq|mariadb|mysql|mailhog|openssl)(@\d\..*)?\s"</summary>
<pre>dnsmasq 2.86
nginx 1.21.5
openssl@1.1 1.1.1m
php 8.1.1
php@7.4 7.4.27
php@8.0 8.0.14</pre>
</details>
<details>
<summary>brew outdated</summary>
<pre></pre>
</details>
<details>
<summary>brew tap</summary>
<pre>homebrew/core
homebrew/services
shivammathur/php</pre>
</details>
<details>
<summary>php -v</summary>
<pre>PHP 8.0.14 (cli) (built: Dec 17 2021 22:21:16) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.14, Copyright (c) Zend Technologies
    with Zend OPcache v8.0.14, Copyright (c), by Zend Technologies</pre>
</details>
<details>
<summary>which -a php</summary>
<pre>/usr/local/bin/php
/usr/local/bin/php</pre>
</details>
<details>
<summary>php --ini</summary>
<pre>Configuration File (php.ini) Path: /usr/local/etc/php/8.0
Loaded Configuration File:         /usr/local/etc/php/8.0/php.ini
Scan for additional .ini files in: /usr/local/etc/php/8.0/conf.d
Additional .ini files parsed:      /usr/local/etc/php/8.0/conf.d/error_log.ini,
/usr/local/etc/php/8.0/conf.d/ext-opcache.ini,
/usr/local/etc/php/8.0/conf.d/php-memory-limits.ini</pre>
</details>
<details>
<summary>nginx -v</summary>
<pre>nginx version: nginx/1.21.5</pre>
</details>
<details>
<summary>curl --version</summary>
<pre>curl 7.77.0 (x86_64-apple-darwin21.0) libcurl/7.77.0 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.42.0
Release-Date: 2021-05-26
Protocols: dict file ftp ftps gopher gophers http https imap imaps ldap ldaps mqtt pop3 pop3s rtsp smb smbs smtp smtps telnet tftp 
Features: alt-svc AsynchDNS GSS-API HSTS HTTP2 HTTPS-proxy IPv6 Kerberos Largefile libz MultiSSL NTLM NTLM_WB SPNEGO SSL UnixSockets</pre>
</details>
<details>
<summary>php --ri curl</summary>
<pre>curl

cURL support => enabled
cURL Information => 7.80.0
Age => 9
Features
AsynchDNS => Yes
CharConv => No
Debug => No
GSS-Negotiate => No
IDN => Yes
IPv6 => Yes
krb4 => No
Largefile => Yes
libz => Yes
NTLM => Yes
NTLMWB => Yes
SPNEGO => Yes
SSL => Yes
SSPI => No
TLS-SRP => Yes
HTTP2 => Yes
GSSAPI => Yes
KERBEROS5 => Yes
UNIX_SOCKETS => Yes
PSL => No
HTTPS_PROXY => Yes
MULTI_SSL => Yes
BROTLI => Yes
Protocols => dict, file, ftp, ftps, gopher, gophers, http, https, imap, imaps, ldap, ldaps, mqtt, pop3, pop3s, rtmp, rtsp, scp, sftp, smb, smbs, smtp, smtps, telnet, tftp
Host => x86_64-apple-darwin21.1.0
SSL Version => (SecureTransport) OpenSSL/1.1.1m
ZLib Version => 1.2.11
libSSH Version => libssh2/1.10.0

Directive => Local Value => Master Value
curl.cainfo => no value => no value</pre>
</details>
<details>
<summary>~/.composer/vendor/laravel/valet/bin/ngrok version</summary>
<pre>ngrok version 2.3.40</pre>
</details>
<details>
<summary>ls -al ~/.ngrok2</summary>
<pre>ls: /Users/xxx/.ngrok2: No such file or directory</pre>
</details>
<details>
<summary>brew info nginx</summary>
<pre>nginx: stable 1.21.5 (bottled), HEAD
HTTP(S) server and reverse proxy, and IMAP/POP3 proxy server
https://nginx.org/
/usr/local/Cellar/nginx/1.21.5 (26 files, 2.2MB) *
  Poured from bottle on 2022-01-04 at 18:53:24
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/nginx.rb
License: BSD-2-Clause
==> Dependencies
Required: openssl@1.1, pcre
==> Options
--HEAD
	Install HEAD version
==> Caveats
Docroot is: /usr/local/var/www

The default port has been set in /usr/local/etc/nginx/nginx.conf to 8080 so that
nginx can run without sudo.

nginx will load all files in /usr/local/etc/nginx/servers/.

To restart nginx after an upgrade:
  brew services restart nginx
Or, if you don't want/need a background service you can just run:
  /usr/local/opt/nginx/bin/nginx -g daemon off;
==> Analytics
install: 29,182 (30 days), 115,783 (90 days), 495,406 (365 days)
install-on-request: 29,129 (30 days), 115,573 (90 days), 494,307 (365 days)
build-error: 5 (30 days)</pre>
</details>
<details>
<summary>brew info php</summary>
<pre>php: stable 8.1.1 (bottled), HEAD
General-purpose scripting language
https://www.php.net/
/usr/local/Cellar/php/8.1.1 (512 files, 79.9MB)
  Poured from bottle on 2022-01-04 at 18:49:14
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/php.rb
License: PHP-3.01
==> Dependencies
Build: httpd, pkg-config
Required: apr, apr-util, argon2, aspell, autoconf, curl, freetds, gd, gettext, gmp, icu4c, krb5, libpq, libsodium, libzip, oniguruma, openldap, openssl@1.1, pcre2, sqlite, tidy-html5, unixodbc
==> Options
--HEAD
	Install HEAD version
==> Caveats
To enable PHP in Apache add the following to httpd.conf and restart Apache:
    LoadModule php_module /usr/local/opt/php/lib/httpd/modules/libphp.so

    <FilesMatch \.php$>
        SetHandler application/x-httpd-php
    </FilesMatch>

Finally, check DirectoryIndex includes index.php
    DirectoryIndex index.php index.html

The php.ini and php-fpm.ini file can be found in:
    /usr/local/etc/php/8.1/

To restart php after an upgrade:
  brew services restart php
Or, if you don't want/need a background service you can just run:
  /usr/local/opt/php/sbin/php-fpm --nodaemonize
==> Analytics
install: 98,319 (30 days), 292,314 (90 days), 754,148 (365 days)
install-on-request: 78,447 (30 days), 240,503 (90 days), 694,133 (365 days)
build-error: 89 (30 days)</pre>
</details>
<details>
<summary>brew info openssl</summary>
<pre>openssl@3: stable 3.0.1 (bottled) [keg-only]
Cryptography and SSL/TLS Toolkit
https://openssl.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/openssl@3.rb
License: Apache-2.0
==> Dependencies
Required: ca-certificates
==> Caveats
A CA file has been bootstrapped using certificates from the system
keychain. To add additional certificates, place .pem files in
  /usr/local/etc/openssl@3/certs

and run
  /usr/local/opt/openssl@3/bin/c_rehash

openssl@3 is keg-only, which means it was not symlinked into /usr/local,
because macOS provides LibreSSL.

==> Analytics
install: 101,745 (30 days), 272,599 (90 days), 316,100 (365 days)
install-on-request: 72,981 (30 days), 215,059 (90 days), 250,128 (365 days)
build-error: 4,626 (30 days)</pre>
</details>
<details>
<summary>openssl version -a</summary>
<pre>LibreSSL 2.8.3
built on: date not available
platform: information not available
options:  bn(64,64) rc4(16x,int) des(idx,cisc,16,int) blowfish(idx) 
compiler: information not available
OPENSSLDIR: "/private/etc/ssl"</pre>
</details>
<details>
<summary>openssl ciphers</summary>
<pre>ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA:ECDHE-ECDSA-AES256-SHA:DHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-CHACHA20-POLY1305:GOST2012256-GOST89-GOST89:DHE-RSA-CAMELLIA256-SHA256:DHE-RSA-CAMELLIA256-SHA:GOST2001-GOST89-GOST89:AES256-GCM-SHA384:AES256-SHA256:AES256-SHA:CAMELLIA256-SHA256:CAMELLIA256-SHA:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES128-SHA:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-CAMELLIA128-SHA256:DHE-RSA-CAMELLIA128-SHA:AES128-GCM-SHA256:AES128-SHA256:AES128-SHA:CAMELLIA128-SHA256:CAMELLIA128-SHA:ECDHE-RSA-RC4-SHA:ECDHE-ECDSA-RC4-SHA:RC4-SHA:RC4-MD5:ECDHE-RSA-DES-CBC3-SHA:ECDHE-ECDSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:DES-CBC3-SHA</pre>
</details>
<details>
<summary>sudo nginx -t</summary>
<pre>nginx: the configuration file /usr/local/etc/nginx/nginx.conf syntax is ok
nginx: configuration file /usr/local/etc/nginx/nginx.conf test is successful</pre>
</details>
<details>
<summary>which -a php-fpm</summary>
<pre></pre>
</details>
<details>
<summary>/usr/local/opt/php/sbin/php-fpm -v</summary>
<pre>PHP 8.1.1 (fpm-fcgi) (built: Dec 17 2021 22:38:16)
Copyright (c) The PHP Group
Zend Engine v4.1.1, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.1, Copyright (c), by Zend Technologies</pre>
</details>
<details>
<summary>sudo /usr/local/opt/php/sbin/php-fpm -y /usr/local/etc/php/8.0/php-fpm.conf --test</summary>
<pre>[05-Jan-2022 20:35:28] NOTICE: configuration file /usr/local/etc/php/8.0/php-fpm.conf test is successful</pre>
</details>
<details>
<summary>ls -al ~/Library/LaunchAgents | grep homebrew</summary>
<pre>-rw-r--r--   1 xxx  staff   555  5 Jan 10:59 homebrew.mxcl.mariadb@10.5.plist
-rw-r--r--   1 xxx  staff   577  5 Jan 20:29 homebrew.mxcl.php.plist</pre>
</details>
<details>
<summary>ls -al /Library/LaunchAgents | grep homebrew</summary>
<pre></pre>
</details>
<details>
<summary>ls -al /Library/LaunchDaemons | grep homebrew</summary>
<pre>-rw-r--r--   1 root  admin   593  5 Jan 20:29 homebrew.mxcl.dnsmasq.plist
-rw-r--r--   1 root  admin   484  5 Jan 20:32 homebrew.mxcl.nginx.plist
-rw-r--r--   1 root  admin   585  5 Jan 20:32 homebrew.mxcl.php@8.0.plist</pre>
</details>
<details>
<summary>ls -al /Library/LaunchDaemons | grep "com.laravel.valet."</summary>
<pre></pre>
</details>
<details>
<summary>ls -aln /etc/resolv.conf</summary>
<pre>lrwxr-xr-x  1 0  0  22  7 Dec 23:39 /etc/resolv.conf -> ../var/run/resolv.conf</pre>
</details>
<details>
<summary>cat /etc/resolv.conf</summary>
<pre>#
# macOS Notice
#
# This file is not consulted for DNS hostname resolution, address
# resolution, or the DNS query routing mechanism used by most
# processes on this system.
#
# To view the DNS configuration used by this system, use:
#   scutil --dns
#
# SEE ALSO
#   dns-sd(1), scutil(8)
#
# This file is automatically generated.
#
search Home
nameserver 192.168.0.1
nameserver fd7e:abbc:1a70:0:8275:1fff:fe3d:64e8</pre>
</details>
<details>
<summary>ifconfig lo0</summary>
<pre>lo0: flags=8049<UP,LOOPBACK,RUNNING,MULTICAST> mtu 16384
	options=1203<RXCSUM,TXCSUM,TXSTATUS,SW_TIMESTAMP>
	inet 127.0.0.1 netmask 0xff000000 
	inet6 ::1 prefixlen 128 
	inet6 fe80::1%lo0 prefixlen 64 scopeid 0x1 
	nd6 options=201<PERFORMNUD,DAD></pre>
</details>
<details>
<summary>sh -c 'echo "------\n/usr/local/etc/nginx/valet/valet.conf\n---\n"; cat /usr/local/etc/nginx/valet/valet.conf | grep -n "# valet loopback"; echo "\n------\n"'</summary>
<pre>------
/usr/local/etc/nginx/valet/valet.conf
---

3:    #listen VALET_LOOPBACK:80; # valet loopback

------</pre>
</details>
<details>
<summary>sh -c 'for file in ~/.config/valet/dnsmasq.d/*; do echo "------\n~/.config/valet/dnsmasq.d/$(basename $file)\n---\n"; cat $file; echo "\n------\n"; done'</summary>
<pre>------
~/.config/valet/dnsmasq.d/tld-test.conf
---

address=/.test/127.0.0.1
listen-address=127.0.0.1

------</pre>
</details>
<details>
<summary>sh -c 'for file in ~/.config/valet/nginx/*; do echo "------\n~/.config/valet/nginx/$(basename $file)\n---\n"; cat $file | grep -n "# valet loopback"; echo "\n------\n"; done'</summary>
<pre>------

------
~/.config/valet/nginx/blankapp.test
---

3:    #listen 127.0.0.1:80; # valet loopback
10:    #listen 127.0.0.1:443 ssl http2; # valet loopback
54:    #listen 127.0.0.1:60; # valet loopback

------</pre>
</details>

**Comment by ihorvorotnov** [comment]
Happens to me from time to time (like, 1-2 times per week) on M1 Pro and macOS Monterey. When this happens, check Activity Monitor > CPU, in my case dnsmasq takes up ~100% CPU for some reason. Force-quitting the process fixes the issue.

**Comment by estin92** [comment, marked as answer]
Thanks @ihorvorotnov, will look into that.

This morning I decided to erase the MacBook and start over, installing valet on a clean machine before adding software and packages before determining if anything delayed the DNS look up.

It would appear that the Cisco AnyConnect VPN client is to blame. I had valet running at full speed until the software was installed, after which the lag started again. Uninstalling the software then reduced the DNS look up time. I've reached out to Cisco to see if there's an updated package for ARM.

**Reply by Frulko** [reply]
I have the same issue of slow DNS look-up from dnsmasq. 

I'm not on M1 Macbook but on classic Intel. I have same issue.

I didn't expected that Cisco AnyConnect VPN could be the origin of that problem. Thanks for finding that. 
My only solution before your comment was to add by hand each domain inside `/etc/hosts`

Thanks a lot, uninstall Cisco AnyConnect VPN is a really time-saver :)

**Reply by estin92** [reply]
Are you on macOS 12 @Frulko? Wondering if that's the culprit. Glad the tip on Cisco worked for you, too.

**Reply by Frulko** [reply]
Yep I'm on macOS 12, maybe a conflict between Cisco and dnsmasq

**Reply by estin92** [reply]
Thanks, @Frulko I've updated the ticket name to reflect the issue being with macOS 12 as opposed to the M1 chip.

As an aside, I've been doing some more work on getting AnyConnect to work with macOS12 as we rely quite heavily on it to access certain systems.

On installation of AnyConnect, macOS warns of a legacy system extension relating to the Socket Filtering application that comes bundled with the AnyConnect client. Normally, I'd choose to enable this. I have however recently tried ignoring this warning and leaving it blocked. 

I can confirm leaving the filter extension blocked has meant we can access the resources we need through the VPN and valet is running without any lengthy DNS look ups. Not sure this may work for everyone, but might be worth a try?

<img width="780" alt="Screenshot 2022-01-08 at 20 23 34" src="https://user-images.githubusercontent.com/11060245/148658926-29090365-7c28-42c5-b00c-2116f2c16fd6.png">

**Comment by rodrigore** [comment]
Hey @estin92  thanks a lot, I was having the same issue on my M1 Max using Mac OS 13 (ventura)

After installing cisco **anyconnect-macos-4.10.05095-predeploy-k9.dmg** the initial load of every site was really slow and painful. I uninstall that version and tried an old version, and is working without issue.

The version that works for me is **anyconnect-macos-4.8.03036-predeploy-k9** (GitHub don't allow me to attach a zip or dmg file)
